folding-schemes/src/folding/hypernova/lcccs.rs

```diff
@@ -96,7 +96,7 @@ impl<C: CurveGroup> LCCCS<C> {
         w: &Witness<C::ScalarField>,
     ) -> Result<(), Error> {
         // check that C is the commitment of w. Notice that this is not verifying a Pedersen
-        // opening, but checking that the Commmitment comes from committing to the witness.
+        // opening, but checking that the Commitment comes from committing to the witness.
```

Comment:
```suggestion
        // opening, but checking that the commitment comes from committing to the witness.
```

while we are at it, lowercasing here :p

Reply:
Right, [updated it](https://github.com/privacy-scaling-explorations/folding-schemes/compare/6c4191a9449414af8c0db35b817d13415b794837..39e0fc494622615a5398d68b8a287dbc4551a45b), both in lcccs.rs and also in cccs.rs which had a similar sentence with the same uppercase `Commitment` word.